docker/templates/Dockerfile.j2

```
@@ -147,7 +149,24 @@ ADD pipeline/default.conf pipeline/logstash.conf
 RUN chown --recursive logstash:root config/ pipeline/
 # Ensure Logstash gets the correct locale by default.
 ENV LANG={{ locale }} LC_ALL={{ locale }}
-ADD env2yaml/env2yaml /usr/local/bin/
+
+# Copy over the appropriate env2yaml artifact
+COPY env2yaml/env2yaml-amd64 env2yaml/env2yaml-arm64 env2yaml/
+
+RUN env2yamlarch="$({{ arch_command }})"; \
```

While you've got the hood open, I'd also suggest `set -eux` here (at the very least `set -e` so failures propagate correctly):

```suggestion
RUN set -eux; \
  env2yamlarch="$({{ arch_command }})"; \
```